docs/os/windows/windows-tools41.md

```diff
@@ -101,7 +101,7 @@ This will allow you to install the Qubes Windows Tools on Windows 7, 10 and 11 b
 
 	Once the Windows VM boots, a CDROM should appear in the 'My Computer' menu (typically as `D:` or `E:`) with the setup program `qubes-tools-x64.msi` in its main directory.
 
- 4. Install Qubes Windows Tools by starting `qubes-tools-x64.msi` as administrator, optionally selecting the `Xen PV disk drivers`. For installation in a template, you should select `Move user profiles`.
+ 4. Install Qubes Windows Tools by starting `qubes-tools-x64.msi` (logged in as administrator), optionally selecting the `Xen PV disk drivers`. For installation in a template, you should select `Move user profiles`.
```

Comment:
just curious - is it mandatory to be logged in as an admin ? ie. being logged as a standard user and starting the install exe "as administrator" doesn't work ?